benchmark/error/node-error.js

```diff
@@ -3,19 +3,21 @@
 const common = require('../common');
 
 const bench = common.createBenchmark(main, {
-  n: [1e7],
+  n: [1e6],
+  code: ['built-in', 'ERR_HTTP2_STREAM_SELF_DEPENDENCY'],
+  stackTraceLimit: [0, 10],
 }, {
   flags: ['--expose-internals'],
 });
 
-function main({ n }) {
-  const {
-    codes: {
-      ERR_INVALID_STATE,
-    },
-  } = require('internal/errors');
+
+function main({ n, code, stackTraceLimit }) {
+  const Err = code === 'built-in' ? Error : require('internal/errors').codes[code];
+
+  Error.stackTraceLimit = stackTraceLimit;
   bench.start();
+
   for (let i = 0; i < n; ++i)
-    new ERR_INVALID_STATE.TypeError('test');
+    new Err();
```

Comment:
You should assign it to a variable and maybe `assert.ok()` after `bench.end(n)` to prevent dead-code elimination.

Reply:
done